src/Microsoft.Dnx.Tooling/ConsoleCommands/ListConsoleCommand.cs

```diff
@@ -26,6 +26,9 @@ public static void Register(CommandLineApplication cmdApp, ReportsFactory report
                 var details = c.Option("--details",
                     "Show the details of how each dependency is introduced",
                     CommandOptionType.NoValue);
+                var mismatch = c.Option("--mismatch",
+                    "Show the mismatch dependencies.",
```

Comment:
Yeah the option should be `--mismatched`, unless we can come up with a shorter, different word. E.g. `--yucky`.